### PR TITLE
[Showcase Apps] Add sample app for FormCell inline validation

### PR DESCRIPTION
**Pull request by robinkuck**

### CODE REVIEW

**Comment by billfroelich** [comment]
I would suggest we also include helper text as part of the examples and possibly additional form cell controls as well

**Comment by robinkuck** [comment]
@billfroelich good point. I will include all remaining FormCell controls with inline validation to the page + add samples for helper text etc.

File: Showcase_Apps/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ All of the applications in these folders are full working applications.  Please
 
 ## Applications
 * Image Handling
+* FormCell Inline Validation
```
Comment (jitendrakansal) [inline Showcase_Apps/README.md:10]: Can you move "FormCell Inline Validation" before "Image Handling" to have those showcase apps (no extensions) in an alphabetical order?

Comment (robinkuck) [reply Showcase_Apps/README.md:9]: Done

**Comment by robinkuck** [comment]
@billfroelich @jitendrakansal in [60e952f](https://github.com/SAP-samples/cloud-mdk-samples/pull/32/commits/60e952f1f631aabb4ac1e3115b134c24db976f92) I added all formcell controls + sample validation. The only todo left is to replace the screenshots.

File: Showcase_Apps/InlineValidationViewApp/InlineValidationViewApp/Pages/Main.page [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+							"AllowEmptySelection": true,
+							"Caption": "Choose Single",
+							"DataPaging": {
+								"ShowLoadingIndicator": false,
+								"PageSize": 50
+							},
+							"PickerPrompt": "Please select one single item",
+							"HelperText": "HelperText",
+							"IsSelectedSectionEnabled": false,
+							"IsPickerDismissedOnSelection": false,
```
Comment (jitendrakansal) [inline Showcase_Apps/InlineValidationViewApp/InlineValidationViewApp/Pages/Main.page:102]:
```suggestion
							"IsPickerDismissedOnSelection": true,
```

**Comment by jitendrakansal** [comment]
@robinkuck there are some related bugs are being fixed as part of 23.4.1 patch release. Let's wait till then

**Review by jitendrakansal — APPROVED** [review]

**Pull request merged**